Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2410,14 +2392,14 @@ private HandleType GetHandleType(Transform handle)
         {
             for (int i = 0; i < balls.Count; ++i)
             {
-                if (handle == balls[i])
+                if (handle.position == balls[i].position && handle.rotation == balls[i].rotation)
```

Comment:
so had a second look now and I'm not sure I understand why you had to change to the visuals here instead of sticking to the handle object itself? The handle object is a parent of the visual, so it should be fine to use it's transforms and it's also making these checks less error prone as you're comparing the object and not just some arbitrary position / rotation values

Reply:
The main reason for this change was for testing purposes, as the only public way to access handlers was through the ScaleCorners and RotateMidpoints objects, which refer to the corners and balls objects. I can rework ScaleCorners and RotateMidpoints to directly refer to the handler objects themselves though